### ISSUE TITLE
Docs: enable translations of code blocks

### ISSUE THREAD

**Issue by m-aciek** [python/cpython#123407]
# Documentation

Code blocks, e.g. in the tutorial, include comments that are complement to the regular content of tutorial articles. An example in tutorial's introduction in [section “Numbers”](https://docs.python.org/3/tutorial/introduction.html#numbers):
```python
>>> 17 / 3  # classic division returns a float
5.666666666666667
>>>
>>> 17 // 3  # floor division discards the fractional part
5
>>> 17 % 3  # the % operator returns the remainder of the division
2
>>> 5 * 3 + 2  # floored quotient * divisor + remainder
17
```

I'd like to propose changing gettext builder configuration by allowing `'literal-block'` [element types translation](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-gettext_additional_targets).

The topic previously was brought up on [doc-sig mailing list](https://mail.python.org/pipermail/doc-sig/2020-January/004190.html) without a conclusion.

### Linked PRs
* gh-123408
* gh-123530
* gh-123531
* gh-123852

**Comment by zware** [comment]
Makes sense to me.  Would you like to submit a PR?

**Comment by m-aciek** [comment]
Sure! Here it is: https://github.com/python/cpython/pull/123408.

**Comment by JulienPalard** [comment]
Damned, I just noticed that, it broke the po syntax checker, the orthograph checker, the grammar checker, and sphinx-lint in python-docs-fr CI :D I'll have to find a way to disable it :D

**Comment by m-aciek** [comment]
Sorry for that! Please see [my comment](https://github.com/sphinx-contrib/sphinx-lint/issues/120#issuecomment-2451012041) in sphinx-lint.

**Comment by rffontenelle** [comment]
@JulienPalard My workaround for this issue is basically disabling 'literal-block', generating POT, updating PO, running all desired checkers, and then undoing this change in po files. Obviously, this workaround cannot be run before committing the changes.

In shell script, this means something like:

```bash
cd cpython/Doc
# comment (removing would work too)
sed -i "/^\s*'literal-block',/s/ '/ #'/" conf.py
# generate new po template files
make gettext
# update PO files from POT
sphinx-intl update --locale-dir path/to/pofiles --pot-dir build/gettext
# run the desired checkers
# finally, undo the changes
git -C path/to/pofiles checkout .
```

p.s.: make gettext is available as of Python 3.12.
